runHarness/WorkloadDrivers/AuctionWorkloadDriver.pm

```diff
@@ -1928,26 +1928,45 @@ sub getWorkloadStatsSummary {
 	my $overallAvgRT = 0;
 
 	my $numPassingAi = 0;
+	my $numTotalAi = 0;
 	my $appInstancesRef = $self->workload->appInstancesRef;
+	$numTotalAi = scalar @$appInstancesRef;
```

Comment:
I think all other parts of the code used $# with +1 to get number of elements, although scalar should work also.